dbt/README.md

```diff
@@ -625,12 +669,160 @@ do so, you have two options:
+  using `dbt run` prior to export. It defaults to `False`, and is only useful in rare cases
+  where the underlying models that comprise the reports have been edited since the last run,
+  typically during the period when a QC report is under active development.
+
+#### Example: Running town close QC reports
+
+We tag the models that comprise our town close QC reports using the `qc_report_town_close`
+tag, and we filter them for a specific township code (like "70") and tax year during export.
+
+Here's an example of how to export those models for a township code defined by `$TOWNSHIP_CODE`
+and a tax year defined by `$TAXYR`: 
+
+```
+python3 scripts/export_models.py --select tag:qc_report_town_close --where "taxyr = '$TAXYR' and township_code = '$TOWNSHIP_CODE'"
+```
+
+The script will output the reports to the `dbt/export/output/` directory, and will print the
+names of the reports that it exports during execution.
+
+#### Example: Running the AHSAP change in value QC report
+
+We define the AHSAP change in value QC report using one model, `qc.vw_change_in_ahsap_values`,
+which we filter for a specific township name (like "Hyde Park") and tax year during export.
+
+Here's an example of how to export that model for a township name defined by `$TOWNSHIP_NAME`
+and a tax year defined by `$TAXYR`:
+
+```
+python3 scripts/export_models.py --select qc.vw_change_in_ahsap_values --where "taxyr = '$TAXYR' and township_name = '$TOWNSHIP_NAME'"
+```
+
+The script will output the reports to the `dbt/export/output/` directory, and will print the
+names of the reports that it exports during execution.
+
+#### Adding QC reports
+
+Since QC reports are built on top of models, adding a new QC report can be as simple
+as adding a new model and exporting it using the [`export_models`
+script](https://github.com/ccao-data/data-architecture/blob/master/dbt/scripts/export_models.py).
+You should default to adding your model to the `qc` schema and subdirectory, unless there is
+a good reason to define it elsewhere. For details on how to add a model, see
+[➕ How to add a new model](#-how-to-add-a-new-model).
+
+There are a number of configuration options that allow you to control the format of your
+model during and after export:
+
+* **Tagging**: [Model tags](https://docs.getdbt.com/reference/resource-configs/tags) are not
+  required for QC reports, but they are helpful in cases where we need to export more than one
+  report at a time. For example, Valuations typically requests all of the town close QC
+  reports at the same time, so we tag each model with the `qc_report_town_close` tag such that
+  we can select them all at once when running the `export_models` script using
+  `--select tag:qc_report_town_close`. For consistency, prefer tags that start with the `qc_*`
+  prefix, but beware not to use the `test_qc_*` prefix, which is instead used for [QC
+  tests](#adding-qc-tests).
```

Comment:
The distinction between these two tag formats strikes me as a little bit confusing... Any ideas for a more intuitive tagging scheme?

Reply:
Following up from above, how about something like:

- QC tests --> Data Integrity (DI) tests = `test_di...`
- Non-QC tests --> Integration tests = `test_int...`
- Unit tests (unchanged) = `test_unit...`
- QC reports (unchanged) = `report_qc...`

With the expectation that we may extend the report tag to encompass other types of exports and reporting.

Reply:
I switched to a slightly different naming scheme in [db596f4](https://github.com/ccao-data/data-architecture/pull/568/commits/db596f4e857a68c5713794d2d976254657942f88) that we talked about yesterday, namely consolidating "QC tests" and "non-QC tests" under a unified "data tests" category. Currently working on a follow-up https://github.com/ccao-data/data-architecture/pull/571 that will handle renaming the tags, since that touches a lot of code.